src/components/proofs/ProofButton.tsx

```diff
@@ -32,9 +33,9 @@ export default function ({
   children,
   disabled,
   ...rest
-}: React.HTMLAttributes<HTMLButtonElement>) {
+}: HTMLAttributes<HTMLButtonElement>) {
   return (
-    <button className={button(disabled)} disabled={disabled} {...rest}>
+    <button className={button(!!disabled)} disabled={disabled} {...rest}>
```

Comment:
Can you simply set it as `button(disabled)`?

Reply:
Actually we don't need it :)